src/natcap/invest/urban_nature_access.py

```diff
@@ -167,27 +171,113 @@
                         '"weight = 0.75 * (1-(pixel_dist / search_radius)^2)"'
                     ),
                 },
+                KERNEL_LABEL_POWER: {
+                    'display_name': 'Power',
+                    'description': gettext(
+                        'Contributions to a greenspace pixel decrease '
+                        'according to a user-defined negative power function '
+                        'of the form "weight = pixel_dist^beta", where beta '
+                        'is expected to be negative and defined by the user.'
+                    ),
+                },
             },
             'about': (
                 'Pixels within the search radius of a greenspace pixel '
                 'have a distance-weighted contribution to a greenspace '
                 'pixel according to the selected decay function.'),
         },
+        'search_radius_mode': {
+            'name': 'search radius mode',
+            'type': 'option_string',
+            'required': True,
+            'about': gettext(
+                'The type of search radius to use.'
+            ),
+            'options': {
+                RADIUS_OPT_UNIFORM: {
+                    'display_name': 'Uniform radius',
+                    'description': gettext(
+                        'The search radius is the same for all greenspace '
+                        'types.'),
+                },
+                RADIUS_OPT_GREENSPACE: {
+                    'display_name': 'Radius defined per greenspace class',
+                    'description': gettext(
+                        'The search radius is defined for each distinct '
+                        'greenspace LULC classification.'),
+                },
+                RADIUS_OPT_POP_GROUP: {
+                    'display_name': 'Radius defined per population group',
+                    'description': gettext(
+                        'The search radius is defined for each distinct '
+                        'population group.'),
+                },
+            },
+        },
+        'aggregate_by_pop_group': {
+            'type': 'boolean',
+            'name': 'Aggregate by population groups',
+            'required': False,
+            'about': gettext(
+                'Whether to aggregate statistics by population group '
+                'within each administrative unit. If selected, population '
+                'groups will be read from the fields of the user-defined '
+                'admin units vector. This option is implied if the search '
+                'radii are defined by population groups.'
```

Comment:
I think here I assumed that 'requirement' only means that the option needs to be present in the args dict, not that the value must be true.  When `search_radius_mode == "{RADIUS_OPT_POP_GROUP}`, the option is assumed to be present in the args dict and is assumed to be `True`, since the option is implied.  Does that make sense?

I agree, this is a little funky to try to express clearly!